### PR TITLE
examples: Sandbox cleanups

### PR DESCRIPTION
**Pull request by phlax**
Commit Message: examples: Sandbox cleanups
Additional Description:

As im reviewing sandbox PRs im seeing the same unnecessary configs repeated

This PR will clean up the existing configs, which should hopefully prevent this.

Specifically it removes

- `expose` directives from docker-compose
- unnecessary `port` directives
- unused envoy admin interfaces 
- unnecessary network aliases

Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue] touch #13060 
[Optional Deprecated:]

### CODE REVIEW

**Comment by phlax** [comment]
im just going through the docs.

If they mention the availability of an admin ui, im adding some docs/tests for that...

File: docs/root/start/sandboxes/cache.rst [hunk 1]
```diff
@@ -8,9 +8,9 @@ In this example, we demonstrate how HTTP caching can be utilized in Envoy by usi
 The setup of this sandbox is based on the setup of the :ref:`Front Proxy sandbox <install_sandboxes_front_proxy>`.
 
 All incoming requests are routed via the front Envoy, which acts as a reverse proxy sitting on
-the edge of the ``envoymesh`` network. Ports ``8000`` and ``8001`` are exposed by docker
+the edge of the ``envoymesh`` network. Port ``8000`` is exposed by docker
 compose (see :repo:`/examples/cache/docker-compose.yaml`) to handle ``HTTP`` calls
-to the services, and requests to ``/admin`` respectively. Two backend services are deployed behind the front Envoy, each with a sidecar Envoy.
```
Comment (phlax) [inline docs/root/start/sandboxes/cache.rst:13]: @yosrym93 could you confirm that there are no cache-specific stats to show the user ?

if there are ill add back and add docs/test

Comment (yosrym93) [reply docs/root/start/sandboxes/cache.rst:13]: No. The admin interface and its docs were copied as-is from the front proxy example. Feel free to remove this.

Comment (phlax) [reply docs/root/start/sandboxes/cache.rst:13]: if there are some cache-specific stats, i think we should show here

i couldnt see any - but i wasnt certain

Comment (yosrym93) [reply docs/root/start/sandboxes/cache.rst:13]: Do `setResponseFlag` and `setResponseCodeDetails` @ https://github.com/envoyproxy/envoy/blob/master/source/extensions/filters/http/cache/cache_filter.cc#L454 set any admin stats?

Comment (phlax) [reply docs/root/start/sandboxes/cache.rst:13]: not certain - but i guess i could be able to grep for some non-zero returned from cache code

ill have a look further

Comment (toddmgreer) [reply docs/root/start/sandboxes/cache.rst:13]: I don't know anything about the admin interface. CacheFilter doesn't yet report stats (https://github.com/envoyproxy/envoy/issues/13413).

Comment (phlax) [reply docs/root/start/sandboxes/cache.rst:13]: ah great, thanks

ill remove for now, and we can revisit once the filter is reporting stats

**Comment by phlax** [comment]
i have a load more cleanups i would like to do, but i think this PR should be ready to land, and we can deal with those after.

**Review by mattklein123 — APPROVED** [review]
Nice :)

**Pull request merged**